.changeset/purple-onions-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+"effect": patch
+---
+
+support `Refinement` in `Preficate.tuple` and `Predicate.struct`
```

Comment:
```suggestion
support `Refinement` in `Predicate.tuple` and `Predicate.struct`
```

Reply:
Done, also I've marked this change as minor